### ISSUE TITLE
ask about the training

### ISSUE THREAD

**Issue by Wangt-CN** [JunweiLiang/DualAttentionNetwork#1]
Hi, your work is amazing, and i really want to realize your result.
But when i train for one progress bar（probably 60 epoch，34920step), the recall@1 is just 0.01.
So can you tell me the accu/map in first 10 hours' training, i want to know if i have something wrong

Thank you very much!!

**Comment by JunweiLiang** [comment]
When I was training this, I would look at the median rank. The t2i or i2t median rank will drop rapidly from 500 to 200 then to ~50 in the first three validations. After 60 epoch it should have t2i recall@1 ~0.30, then following the paper I lower the learning rate and train for 40 epoch. Also, I observed that ONLY adadelta optimizer works. Somehow adam or momentum never improve the performance.
